### DISCUSSION TITLE
token

### DISCUSSION THREAD

**Discussion by ElpeAJ** [community/community#53925] [Apps, API and Webhooks] [answered]
### Select Topic Area

Question

### Body

Hi, I copied a token on my github account that I use with my github username to git push on my ALX assign
When I put that token in, i now get this message : 'https://github.com': ElpeAJ
Password for 'https://ElpeAJ@github.com': 
remote: Support for password authentication was removed on August 13, 2021.
remote: Please see https://docs.github.com/en/get-started/getting-started-with-git/about-remote-repositories#cloning-with-https-urls for information on currently recommended modes of authentication.
fatal: Authentication failed for 'https://github.com/ElpeAJ/alx-low_level_programming.git/'

When I go on the link, there is a bunch of information but nothing really straightforward to create a new token or figure out what the issue is. Please help out Thank you

**Comment by airtower-luna** [comment, marked as answer]
The message about password authentication seems to appear whenever what you enter isn't a valid token. You mentioned already having a token, I couldn't say if anything went wrong with copy & paste or if there's an issue with the token itself.

See [Creating a personal access token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) for how to create a new token. Make sure to select the **repo** scope (for repository access, like pushing) and if you use Actions also **workflow** (required to modify workflow files).

**Comment by ghostinhershell** [comment]
Hey there!  👋

Thanks for posting in the GitHub Community, @ElpeAJ! We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category. The Accessibility category is a place for our community to discuss and provide feedback on the digital accessibility of GitHub products. Digital accessibility means that GitHub tools, and technologies, are designed and developed so that people with disabilities can use them.

I've gone ahead and moved this to the correct category for you. Good luck!
